cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -9,8 +9,10 @@
 import ddt
 
 from django.test import RequestFactory
+from xmodule.course_module import CourseSummary
 
-from contentstore.views.course import _accessible_courses_list, _accessible_courses_list_from_groups, AccessListFallback
+from contentstore.views.course import _accessible_courses_list, _accessible_courses_list_from_groups, \
+    AccessListFallback, get_courses_accessible_to_user, _staff_accessible_course_list
```

Comment:
Nit: Usually we prefer to wrap things in `()` to manage multiple lines, rather than using `\`